WebView2 landing update (#679)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/index.md

   :::column-end:::
   :::column span="1":::
      **Fixed distribution** \(coming soon\)  
      Choose to package the Chromium bits in your application.  
   :::column-end:::
   :::column span="1":::
      **Incremental adoption**  
      Add web components piece by piece to your application.  
   :::column-end:::
:::row-end:::

## Getting Started  

To build and test your application using the WebView2 control, you need to have both [Microsoft Edge (Chromium)](https://www.microsoftedgeinsider.com/download) and the [WebView2 SDK](https://aka.ms/webviewnuget) installed.  Select one of the following options to get started.  

*   [Getting Started with Win32 C/C++](./gettingstarted/win32.md)  
*   [Getting Started with WPF](./gettingstarted/wpf.md)  
*   [Getting Started with WinForms](./gettingstarted/winforms.md)  

The [WebView2 Samples](https://github.com/MicrosoftEdge/WebView2Samples) repository contains samples that demonstrate all of the WebView2 SDKs features and any API usage patterns.  As the Microsoft Edge WebView team adds more features to the WebView2 SDK, the Microsoft Edge WebView team updates the sample applications.  

> *   [Win32 C/C++ Sample]()  

Reviewer: Please add the missing links.

With WebView2, developers can embed web code in parts of their native application or build the entire native application within a single WebView. View our guides to [Get Started](index.md#getting-started)!

> [!NOTE]
> The WebView2 Preview is intended for early prototyping and to gather feedback from the developer community to helps us shape the future stable API. There will be [breaking changes](releasenotes.md), so it should NOT be used for production apps.

## Hybrid Application Approach

Developers often have to chose between building a web application or a native application. This decision hinges on the trade-off between Reach and Power.

![webnative](./media/WebView2/webnative.PNG)

Web applications allow for a broad reach. Web developers can reuse most, if not all of their code, across all different platforms. Native applications, however, can utilize the capabilities of the entire native platform.

Hybrid applications allow developers to enjoy the best of both worlds. Hybrid application developers benefit from the ubiquity and strength of the web platform as well as the power and full capabilities of the native platform.

## WebView2 Benefits 

> ![webviewreasons](./media/WebView2/webviewreasons.PNG)

Reviewer 2: There are differences between the icon titles and the text below. For example, Fixed vs. Fixed distribution. Can you review please?

Reviewer 3: Josh is going to finalize these two titles in his distribution doc - will fix when his change is complete

> 1. **Web Ecosystem & Skillset:** utilize the entire web platform, libraries, tooling, and talent that exists within the web ecosystem

Reviewer 2: ```suggestion
1. Web ecosystem & skillset: utilize the entire web platform, libraries, tooling, and talent that exists within the web ecosystem.
```

Reviewer 2: To follow the MS Style Guidelines, we should not use bold for emphasis.

> 2. **Rapid Innovation:** web development allows for faster deployment and iteration

Reviewer 2: ```suggestion
2. Rapid Innovation: web development allows for faster deployment and iteration.
```

> 3. **Windows 7, 8, 10 Support:** support for a consistent user experience across Windows 7, 8, and 10 

Reviewer 2: ```suggestion
3. Windows 7, 8, 10 support: support for a consistent user experience across Windows 7, 8, and 10.
```

> 4. **Native Capabilities:** access the full set of Native APIs.

Reviewer 2: ```suggestion
4. Native Capabilities: access the full set of Native APIs.
```

> 5. **Code Sharing:** adding web code to your codebase allows for increased portability across multiple platforms 

Reviewer 2: ```suggestion
5. Code sharing: adding web code to your codebase allows for increased re-use across multiple platforms.
```

> 6. **Microsoft Support:** When WebView2 GA’s it will have the full backing of Microsoft in terms of support and feature requests

Reviewer 2: ```suggestion
6. Microsoft support: When WebView2 GA’s, Microsoft will provide support and add new feature requests.
```

> 7. **Evergreen Distribution:** rely on an up to date version of Chromium with regular and consistent platform updates and security patches 

Reviewer 2: ```suggestion
7. Evergreen distribution: rely on an up-to-date version of Chromium with regular platform updates and security patches.
```

> 8. **Fixed Distribution (Coming Soon):** choice to package the Chromium bits in your application 

Reviewer 2: ```suggestion
8. Fixed distribution (coming soon): choice to package the Chromium bits in your application.
```

> 9. **Incremental Adoption:** add web components piece by piece into your application

Reviewer 2: ```suggestion
9. Incremental adoption: add web components piece by piece to your application.
```

## Getting Started

> To build and test your application using the WebView2 control, you need to have both [Microsoft Edge (Chromium)](https://www.microsoftedgeinsider.com/download/) and the [WebView2 SDK](https://aka.ms/webviewnuget)installed. Select one of the options below to get started! 

Reviewer 2: ```suggestion
To build and test your application using the WebView2 control, you need to have both [Microsoft Edge (Chromium)](https://www.microsoftedgeinsider.com/download/) and the [WebView2 SDK](https://aka.ms/webviewnuget) installed. Select one of the options below to get started.
```

- [Getting Started with Win32 C/C++]()
- [Getting Started with WPF]()

> - [Getting Started with WinForms]()

Reviewer 2: The links are missing.

Reviewer: Updated

> The [WebView2 Samples](https://github.com/MicrosoftEdge/WebView2Samples) repository contains samples that demonstrate all of the WebView2 SDK's features and their API use patterns. As we add more features to the WebView2 SDK, we will regularly update our sample applications.

Reviewer 2: ```suggestion
The [WebView2 Samples](https://github.com/MicrosoftEdge/WebView2Samples) repository contains samples that demonstrate all of the WebView2 SDK's features and their API usage patterns. As we add more features to the WebView2 SDK, we will update our sample applications.
```

- [Win32 C/C++ Sample]()
- [WPF Sample]()

> - [WinForms Sample]()

Reviewer 2: The links are missing.

Reviewer: Updated

## Next Steps

> For more thorough information on how to build and deploy WebView2 application checkout our conceptual documentation and how-to guides.

Reviewer 2: ```suggestion
For more detailed information on how to build and deploy WebView2 applications, checkout our conceptual documentation and how-to guides.
```

### Concepts
1. [WebView2 SDK and Microsoft Edge Versioning]()
2. [Distributing WebView2 Applications]()

> 3. [Utilizing and Disabling Microsoft Edge Browser Features in WebView2]()

Reviewer 2: TODO: add links to published docs when they go live.

Reviewer: Updated.

### How-To Guides
1. [Debugging WebView2 with DevTools and Visual Studio Script Debugging]()

> 2. [Automating and Debugging WebView2 with Microsoft EdgeDriver]()

Reviewer 2: TODO: add links to published docs when they go live.

Reviewer: Updated.

## Supported Platforms

> A developer preview is available for Win32 C/C++, .NET Framework 4.6.2 or later, .NET Core 3.0 or later, [WinUI 3.0](https://docs.microsoft.com/uwp/toolkits/winui3/) on Windows 10, Windows 8.1, Windows 8, Windows 7, Windows Server 2016, Windows Server 2012/2012R2, and Windows Server 2008 R2. 

Reviewer 2: ```suggestion
A developer preview is available for: 

* Win32 C/C++.
* .NET Framework 4.6.2 or later.
* .NET Core 3.0 or later.
* [WinUI 3.0](https://docs.microsoft.com/uwp/toolkits/winui3/) on Windows 10, Windows 8.1, Windows 8, Windows 7, Windows Server 2016, Windows Server 2012/2012R2, and Windows Server 2008 R2. 
```

> ## Feedback

Reviewer 2: ```suggestion
## Getting in touch with the WebView2 team
```

> Help us build a richer WebView2 experience by sharing your feedback! Visit our [feedback repo](https://aka.ms/webviewfeedback) to submit feature requests or bug reports. It’s also a good place to search for known issues.

Reviewer 2: ```suggestion
Help us build a richer WebView2 experience by sharing your feedback. Visit our [feedback repo](https://aka.ms/webviewfeedback) to submit feature requests or bug reports. It’s also a good place to search for known issues.
```
